client/src/app/site/motions/modules/motion-list/components/motion-list/motion-list.component.ts

```diff
@@ -436,4 +445,43 @@ export class MotionListComponent extends ListViewBaseComponent<ViewMotion, Motio
             }
         });
     }
+
+    /**
+     * Checks if there are at least categories, motion-blocks or tags the user can select.
+     */
+    public checkMetaInfo(): void {
+        if (
+            (this.isCategoryAvailable() || this.isMotionBlockAvailable() || this.isTagAvailable()) &&
+            !this.displayedColumnsDesktop.includes('state')
+        ) {
+            this.displayedColumnsDesktop = this.displayedColumnsDesktop.concat('state');
```

Comment:
You should remove it, if none of this is available (e.g. after deletion of the last category/block/tag)